.travis/before_install.sh

```diff
@@ -39,21 +39,15 @@ cd /tmp || die
 [ $TRAVIS_OS_NAME != linux ] || {    
     sudo apt-get update || die
 
-    sudo -H pip install --upgrade pip || die
     pip install --upgrade pip || die
```

Comment:
This line seems to upgrade the system wide `pip`. Does this require sudo privilege?